packages/tasks/src/model-libraries.ts

```diff
@@ -686,6 +686,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"ssr-speech": {
```

Comment:
Can you update both model card ([here](https://huggingface.co/westbrook/SSR-Speech-Mandarin/blob/main/README.md) and [here](https://huggingface.co/westbrook/SSR-Speech-English/blob/main/README.md)) to add `library_name: ssr-speech` in the model card metadata? This way there will be at least 2 models listed in https://huggingface.co/models?other=ssr-speech before we merge this PR.